.env.example

```
@@ -10,4 +10,11 @@ GOOGLE_CLIENT_ID="your-google-client-id"
 GOOGLE_CLIENT_SECRET="your-google-client-secret"
 
 GITHUB_CLIENT_ID="your-github-client-id"
-GITHUB_CLIENT_SECRET="your-github-client-secret"
\ No newline at end of file
+GITHUB_CLIENT_SECRET="your-github-client-secret"
+
+#using gmail
+SMTP_HOST="smtp.gmail.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)